### PR TITLE
README code example

### PR DESCRIPTION
**Pull request by gsquire**
The README had some minor fixes that were needed in order for it to compile.

### CODE REVIEW

**Comment by tikue** [comment]
Thanks so much for catching that! I think we can even remove the first `to_owned` entirely, because `&str`s can be compared to `String`s.

**Comment by killercup** [comment]
I recently saw https://github.com/brson/rust-skeptic which runs test in markdown files likes Readme.mds. Maybe you could use that to prevent cases like these :)

**Comment by tikue** [comment]
@killercup that's amazing, thank you. brson is a hero as usual.

File: README.md [hunk 1]
```diff
@@ -25,14 +25,14 @@ mod hello_service {
...
-        format!("Hello, {}!", s)
+        format!("Hello, {}!", name)
     }
 }
 
 fn main() {
     let server_handle = hello_service::serve("0.0.0.0:0", HelloService, None).unwrap();
     let client = hello_service::Client::new(server_handle.local_addr(), None).unwrap();
-    assert_eq!("Hello, Mom!".into(), client.hello("Mom".into()).unwrap());
+    assert_eq!("Hello, Mom!".to_owned(), client.hello("Mom".to_owned()).unwrap());
```
Comment (tikue) [inline README.md]: Change to `assert_eq!("Hello, Mom!", client.hello("Mom".into()).unwrap());`

**Comment by tikue** [comment]
Accepting this as-is because I don't want to have a broken readme. Thanks again!

**Pull request merged**